src/main/resources/application.yml

```
@@ -26,4 +34,9 @@ logging:
   level:
     org.hibernate.SQL: debug
     org.hibernate.type.descriptor.sql.BasicBinder: trace
-    com.ayu.realty: DEBUG
+    com.ayu.realty: debug
+    org.springframework.security: debug
+
+openai:
+  api:
+    key: ${OPENAI_API_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)